client/screens/__tests__/DropTestScreen.test.tsx

```
@@ -241,6 +241,19 @@ describe('DropTestScreen', () => {
       sealCheckbox.props.onValueChange(true);
     });
   };
+  const recordClosureTime = async (seconds = 4) => {
+    const startButton = findByTestId(renderer, 'timer-start-button');
```

ESLint

[error] Replace `'timer-start-button'` with `"timer-start-button"`

(prettier/prettier)